cmd/juju/application/deploy_test.go

```diff
@@ -1503,7 +1503,7 @@ func (s *DeploySuite) setupNonESMBase(c *gc.C) (series.Base, string) {
 	}
 	defer func() { _ = file.Close() }()
 
-	// Overwrite the metadata.yaml to contain a non EMS series.
+	// Overwrite the metadata.yaml to contain a non EMS corebase.
```

Comment:
Oops

Reply:
damn, caught one of these, missed this one